### PR TITLE
Add kickstart test to test bond interface creation

### PR DESCRIPTION
**Pull request by jkonecny12**
Steps of testing:

* create bond with ``network --device=bond0 --bootproto=static --bondslaves=link --ip=192.168.1.1 --netmask=255.255.252.0 --gateway=192.168.1.2 --nameserver=192.168.1.3 --activate`` ks command
* test ifcfg file in post section

*I'm saving content of the configuration file for the bond0 device to special file for debugging purpose.*

### CODE REVIEW

File: tests/kickstart_tests/network-bond.ks [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+rootpw qweqwe
+shutdown
+
+%packages
+%end
+
+%post
+CONTENT=$(cat /etc/sysconfig/network-scripts/ifcfg-bond0)
+
+printf $CONTENT > /root/IFCFG
```
Comment (vpodzime) [inline tests/kickstart_tests/network-bond.ks]: The same comment here as in the PR #115 -- this should use ``cp``.

File: tests/kickstart_tests/network-bond.ks [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+   echo "ERROR: PREFIX is not present" >> /root/RESULT
+fi
+
+if [[ $CONTENT != *"GATEWAY=192.168.1.2"* ]]; then
+   echo "ERROR: GATEWAY is not present" >> /root/RESULT
+fi
+
+if [[ $CONTENT != *"DNS1=192.168.1.3"* ]]; then
+   echo "ERROR: DNS1 is not present" >> /root/RESULT
+fi
```
Comment (vpodzime) [inline tests/kickstart_tests/network-bond.ks]: And the same comment here about the ``ERROR:`` prefix.

**Comment by vpodzime** [comment]
Looks good to me otherwise.

File: tests/kickstart_tests/network-bond.ks (hunk 1 above)
Comment (jkonecny12) [inline tests/kickstart_tests/network-bond.ks]: Yeah right my fault. I'll change it locally.

File: tests/kickstart_tests/network-bond.ks (hunk 2 above)
Comment (jkonecny12) [inline tests/kickstart_tests/network-bond.ks]: The same response as PR https://github.com/rhinstaller/anaconda/pull/115

**Comment by clumens** [comment]
Looks good to me with Vratislav's comments, thanks!

**Comment by jkonecny12** [comment]
Thank you for your responses.
I'll change it locally and push it.

**Pull request closed without merging**